### PR TITLE
FIX: Missing column Date validation in ledger & subledger

### PR DESCRIPTION
**Pull request by aspangaro**

### CODE REVIEW

File: htdocs/accountancy/bookkeeping/listbyaccount.php [hunk 1]
```diff
@@ -144,6 +152,7 @@
 	't.credit'=>array('label'=>$langs->trans("Credit"), 'checked'=>1),
 	't.lettering_code'=>array('label'=>$langs->trans("LetteringCode"), 'checked'=>1),
 	't.date_export'=>array('label'=>$langs->trans("DateExport"), 'checked'=>1),
+	't.date_validated'=>array('label'=>$langs->trans("DateValidation"), 'checked'=>1),
```
Comment (eldy) [inline htdocs/accountancy/bookkeeping/listbyaccount.php:155]: What is this date ? Is it for a feature of lettering enabled by option ACCOUNTING_ENABLE_LETTERING or by something else ?

Comment (aspangaro) [reply htdocs/accountancy/bookkeeping/listbyaccount.php:155]: No, this does not concern the lettering of accounts. It is related to the FEC format, it is requested to validate the accounting entries and to block them in deletion / modification. Without this functionality, a FEC format can be totally validated

image

Comment (eldy) [reply htdocs/accountancy/bookkeeping/listbyaccount.php:155]: My question was Where and when can we set this date into application ? On which page ? I can't find. Do we have to set an option or beeing in dev lide to be able to set this date ?

Comment (aspangaro) [reply htdocs/accountancy/bookkeeping/listbyaccount.php:155]: This date are set when you export on journal or when you use accoutancy > close menu

**Pull request merged**